packages/sign_in_with_apple/darwin/Classes/SignInWithAppleError.swift

```diff
@@ -107,11 +107,6 @@ public enum SignInWithAppleError {
                 errorCode = "authorization-error/notHandled"
             case .failed:
                 errorCode = "authorization-error/failed"
-#if swift(>=5.5)
```

Comment:
@tp maybe we also need to do `swift(>=5.5) && os(iOS)`, but this seems in some form wrong tbh 😅

Reply:
Might be the best approach for now to keep both building.

But I would expect this to fail in a future Xcode version then.

Reply:
Seems best for now, would proceed with this then.